afdko/Tools/SharedData/FDKScripts/pdfgen.py

```diff
@@ -529,7 +536,7 @@ def setDash(self, array=[], phase=0):
             self._code.append('[%s %s] 0 d' % (array, phase))
```

Comment:
please change line 533 above to:
```python
    def setDash(self, array=None, phase=0):
```
and then add code to handle the `None` case

reference: http://docs.python-guide.org/en/latest/writing/gotchas/